profile.md

```diff
@@ -53,10 +53,11 @@ the [Open Service Broker API specification](spec.md) SHALL take precedence.
 
 - *Service Instance*: An instantiation of a Service offering.
 
-- *Service Binding*: The representation of an association between an
-  Application and a Service Instance. Often, Service Bindings, will
-  contain the credentials that the Application will use to communicate
-  with the Service Instance.
+- *Service Binding*: Represents the request to use a Service Instance. As part
+  of this request there might be a reference to the entity, also known as the
+  Application, that will use the Service Instance. Service Bindings will often
+  contain the credentials that can then be used to communicate with the Service
+  Instance.
```

Comment:
Suggested edit. Can discuss on the call.

*Service Binding*: Represents the intent to use a Service Instance. As part
 of a service binding there might be a reference to the entity, also known as the
 Application, that will use the Service Instance. Service Bindings will often
 contain the credentials and co-ordianates that can then be used to communicate 
 with the Service Instance.

Reply:
Note this is in profile, and we should have a followup PR to sync the term tables from the spec and profile docs.

Reply:
I will PR the profile doc to reference the spec for the definitions. 

WDYT of my proposed text?